pgen2/parse.py

```diff
@@ -92,6 +92,8 @@ def __init__(self, grammar):
         up to the converter function.
         """
         self.grammar = grammar
+        self.rootnode = None  # type: Optional[PNode]
```

Comment:
oh hmmm. I just realized mycpp will generate a class that includes this in the field mask. I change the type in `cpp/pgen2.h` so it is NOT a GC type now. How will the GC react to this? 🤔 Maybe the right thing to do here is give the c++ type a header with `HeapTag::Opaque` so we never scan past the root referenced by the parser...